tests/Doctrine/Tests/DBAL/Performance/TypeConversionPerformanceTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Doctrine\Tests\DBAL\Performance;
+
+use Doctrine\DBAL\Types\Type;
+use Doctrine\Tests\DbalPerformanceTestCase;
+
+/**
+ * Class TypeConversionPerformanceTest
+ * @package Doctrine\Tests\DBAL\Performance
+ * @author Bill Schaller
+ * @group performance
+ */
+class TypeConversionPerformanceTest extends DbalPerformanceTestCase
+{
+    /**
+     * MAXIMUM TIME: 2 seconds
```

Comment:
This is a bit misleading, because it depends on hardware used. I'd rather remove it